consensus/poet/families/sawtooth_validator_registry/validator_registry/processor/handler.py

```diff
@@ -121,6 +122,17 @@ def _set_data(state, address, data):
 
 class ValidatorRegistryTransactionHandler(object):
 
+    # The basename and enclave measurement values we expect to find
+    # in the enclave quote in the attestation verification report.
+    __VALID_BASENAME__ = \
```

Comment:
Where do BASENAME and ENCLAVE_MEASUREMENT come from? Can they change? Can you give some commentary on hardcoding?